### PR TITLE
Created Priority Mananger Bash Script

### PR DESCRIPTION
**Pull request by Mvpl**
@rcapote @rasrocha please review my pull request which includes a script for managing the priorities of gdal and java processes.

### CODE REVIEW

File: prioritymanager.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#this script is suppose to run every 30seconds. The functions in this script change the priority of java
+#(java because snap and wings use java to run the jobs) and gdal processes. When they exceed 7 processes
+#and when they are inferior or equal to 7 processes their priority are changed, using the nice value.
+#The following was added on crontab(using root): 
+#* * * * * /home/coresyf/prioritymanager/prioritymanager.sh
+#* * * * * ( sleep 30 ; /home/coresyf/prioritymanager/prioritymanager.sh )
+
+
+#this function changes the priority of snap processes when they exceed 15 processes
```
Comment (rmcapote) [inline prioritymanager.sh]: 15 processes or 7?

File: prioritymanager.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+prioritySnap () {
+
+    snapIds=$(pgrep java)
+    gdalIds=$(pgrep gdal) #shows all processes by id of this app
+    snapCount=$(pgrep -c java) 
+    gdalCount=$(pgrep -c gdal) #get the number of processes running of an application
+    iterator=0
+
```
Comment (rmcapote) [inline prioritymanager.sh:16]: Only now I have understood that on the function "prioritySnap" you don't need to store "gdalIds" and "gdalCount", these variables are only required in the function "priorityGdal". Right?
Now an optional question: do you see any possible way to refactor the code and make it more simple?

Comment (rmcapote) [reply prioritymanager.sh:16]: :)

**Review by rmcapote — CHANGES_REQUESTED** [review]
@Mvpl the script looks good and we know that it works. See my comment and think if it make sense or not.

**Comment by Mvpl** [comment]
@rmcapote @rasrocha please review my refactoring

**Review by rmcapote — APPROVED** [review]
Ok. It's perfect!

**Pull request merged**